examples/index.json

```diff
@@ -110,12 +110,26 @@
                 "verifyOutput": true,
                 "isLearnByExample": true
             },
+            {
+                "name": "If statement",
+                "url": "if-statement",
+                "verifyBuild": true,
+                "verifyOutput": true,
+                "isLearnByExample": true
+            },
             {
                 "name": "Match statement",
                 "url": "match-statement",
                 "verifyBuild": true,
                 "verifyOutput": true,
                 "isLearnByExample": true
+            },
+            {
+                "name": "Match guard in match statement",
+                "url": "match-guard-in-match-statement",
+                "verifyBuild": true,
+                "verifyOutput": true,
+                "isLearnByExample": true
             }
```

Comment:
Shall we add the details of the `Binding patterns in match statement` BBE here? ATM it is under `Binding Patterns` section. My PR removes that entry from the index.json. Hence we just need to add it here under `Conditional Statements`
```
            {
                "name": "Binding patterns in match statement",
                "url": "binding-patterns-in-match-statement",
                "verifyBuild": true,
                "verifyOutput": true,
                "isLearnByExample": true
            }
```